### PR TITLE
Create abi-technik

### PR DESCRIPTION
**Pull request by zuphilip**
See also http://www.degruyter.com/view/j/abitech

### CODE REVIEW

**Comment by adam3smith** [comment]
(missing file extension)

File: dependent/abi-technik [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="de-DE">
+  <info>
+    <title>ABI Technik</title>
+    <id>http://www.zotero.org/styles/abi-technik</id>
+    <link href="http://www.zotero.org/styles/abi-technik" rel="self"/>
+    <link href="https://www.zotero.org/styles/chicago-fullnote-bibliography" rel="independent-parent"/>
+    <link href="http://www.degruyter.com/view/supplement/s21914664_Autorenhinweise.pdf" rel="documentation"/>
+    <category citation-format="author-date"/>
```
Comment (adam3smith) [inline dependent/abi-technik]: that can't be right --it's either full-note or author-date?

**Comment by zuphilip** [comment]
Huff... Sorry for the errors. Now, it should be okay.

**Comment by adam3smith** [comment]
thanks!

**Comment by rmzelle** [comment]
(just added the language to the title: https://github.com/citation-style-language/styles/commit/ec48b103a2004b19483dc92d8eb3d28a57f74a52 )

Thanks!

**Comment by zuphilip** [comment]
Thank you! BTW I checked the preview at https://www.zotero.org/styles/?q=abi%20technik and it does look to be in English. Is this just taking time or is something else wrong?

**Comment by adam3smith** [comment]
I'd guess that the citeproc that's run on the previews doesn't inherit the locales correctly. It's not something that'll go away with time, no.

**Pull request merged**